enclave/src/c_api/c_api.cc

```diff
@@ -441,7 +441,13 @@ int verify_remote_report_and_set_pubkey(
 }
 
 int add_client_key(uint8_t* data, size_t len, uint8_t* signature, size_t sig_len) {
-    EnclaveContext::getInstance().decrypt_and_save_client_key(data, len, signature, sig_len);
+    if (EnclaveContext::getInstance().decrypt_and_save_client_key(data, len, signature, sig_len))
+      return 0;
+    return -1;
+}
+
+int sync_client_key() {
```

Comment:
See comment for `c_api.h`. Not sure if we need to expose this function to the client.

Reply:
Left the function in here. Took out all other code related to `sync_client_key()`.